addons/web/static/src/report/report.js

```diff
@@ -0,0 +1,47 @@
+/** @odoo-module **/
+
```

Comment:
add a comment to make it clear that it is intended to be run in the report iframe